script.js

```diff
@@ -256,6 +256,31 @@ async function selectPool() {
         alert(`Pool ${poolId} doesn't exist`);
         return;
     }
+    let currentPool = ''
+    try {
+        await setAccountSigner(account, path, publicKey);
+        currentPool = await account.functionCall(
+            lockupAccountId,
+            'get_staking_pool_account_id', 
+            {},
+            '25000000000000');
```

Comment:
Do you need it to be a change-method call? I think you can do:

```suggestion
        currentPool = await account.viewFunction(
            lockupAccountId,
            'get_staking_pool_account_id', 
            {});
```